### PR TITLE
Mobile first into main merge request

### PR DESCRIPTION
**Pull request by glitcher255**
Mobile first into main merge request
Created entire site, phew!

video: https://www.loom.com/share/56e628fc9ebd450d9c18866c4bacb9b0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,8 @@ N/A
 
 Contributions, issues, and feature requests are welcome!
 
+Original design idea by Cindy Shin in Behance.
+
```
Comment (iambenkis) [inline README.md:157]: - [ ] I see your amazing presentation that leave me without word :+1: , but the link should not be in the pull request summary but in the README file  according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-documentation), Kindly create the another section in your README file and drop there your link as you did for live demo link :smile: 
- [ ] Also make sure in your presentation you show each version of your project design. :+1:

**Review by iambenkis — COMMENTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @glitcher255 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:



 ## Required Changes ♻️

- [x] :warning: You did an amazing job :1st_place_medal:  @glitcher255 , but there is an important thing you are to always respect :bow:  to make the code review  process more useful. Is to always merge your branch only after getting an approval from code reviewers :100: 

- [x] Good Job :+1:  creating the section "Featured speakers"  dynamically in JavaScript.

https://github.com/glitcher255/Capstone-project/blob/73b64175a90c6d6aad9b671a05524c2d86dd33c5/index.js#L2-L44

- [x] As mentioned in the requirements  your  pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable. Unfortunately this part below look a bit different that the original design, Kindly make sure the alignment, measurements are identical :100: 
### Your design 

Screenshot from 2023-01-05 19-11-03

### Original design 

Screenshot from 2023-01-05 19-13-37




**[NOTE]  I did not use inline comment to exactly show the place where you should apply the modification. I made my comments clear enough to guide you through the correction process.**

_Check the comments under this block_

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (glitcher255) [reply README.md:157]: @iambenkis This is inside the README.md file, am I missing something here? - it is not in a pull request summary, you grabbed its link in like 156 of the readme file

Comment (glitcher255) [reply README.md:157]: @iambenkis You need to clarify on what "each version" exactly is, what version are you talking about?

File: README.md [hunk 2]
```diff
@@ -132,7 +132,7 @@ N/A
 
 - GitHub: [@githubhandle](https://github.com/glitcher255)
 
-👤 **Jose**
+👤 **Nelson**
```
[the remark this answers is not in the document]
Comment (glitcher255) [reply README.md:135]: Done

**Comment by glitcher255** [comment]
Changes commited to mobile-first branch, sorry for merging this - see https://github.com/glitcher255/Capstone-project/pull/2
for pending changes

**Pull request merged**